ui/pages/swaps/view-on-block-explorer/__snapshots__/view-on-block-explorer.test.js.snap

```diff
@@ -3,7 +3,7 @@
 exports[`ViewOnBlockExplorer renders the component with initial props 1`] = `
 <div>
   <div
-    class="box view-on-block-explorer box--margin-top-6 box--flex-direction-row"
+    class="box view-on-block-explorer box--margin-top-6 box--flex-direction-row box--color-text-default"
   >
     <button>
       View Swap at etherscan.io
```

Comment:
Interesting someone was using the color prop for the Box component already. Will be taken care of in  https://github.com/MetaMask/metamask-extension/pull/15507

Reply:
this has been fixed